src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1563,5 +1563,61 @@ public async Task Http2GetAsync_TrailigPseudo_Throw()
                 await Assert.ThrowsAsync<HttpRequestException>(() => sendTask);
             }
         }
+
+        [Fact]
+        public async Task InboundWindowSize_Exceeded_Throw()
+        {
+            using var semaphore = new SemaphoreSlim(0);
```

Comment:
It probably won't matter, but I think it'd be better to remove the `using` here and just not dispose of the SemaphoreSlim.
1) This is test code, so deterministic cleanup isn't critical.
2) The way the semaphore is being used, dispose would be a nop anyway.
3) Most importantly, I think in a rare circumstance this could actually lead to problems.  Consider if something went wrong in the test and either the client or server failed.  We could then timeout in the CreateClientAndServerAsync, which would throw an exception, and then the semaphore would get disposed.  If the other participant then tried to do its semaphore.WaitAsync or semaphore.Release, it could get an ObjectDisposedException. Now, that's obviously a) super unlikely and b) unlikely to have any real impact, but seems like why bother given (1) and (2).